### DISCUSSION TITLE
query recent window

### DISCUSSION THREAD

**Discussion by basbebe** [asmvik/yabai#948] [Q&A] [answered]
Is it possible to query the recent window?

I would like to write a shell script that swichtes to the recent window, even if on another space.
To make this viable with SIP enabled, I would have to find out the id and the space of the most recent window and I couldn't find it in the json data given by `yabai -m query --windows` and `yabai -m query --spaces`.

**Comment by asmvik** [comment, marked as answer]
is `yabai -m query --windows --window recent` not sufficient?

**Reply by basbebe** [reply]
It seems to not work across spaces, only within the current space.

**Reply by asmvik** [reply]
Hmm I tested this just now, and it does report information across spaces (and displays) for me. It does not work on windows that are in native-fullscreen mode though.

**Reply by basbebe** [reply]
thanks for clarifying!
Seems I was misinterpreting something…
